From c8dc9b585957a9d7c82fff478fe32eb2ae790be8 Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Thu, 23 Nov 2017 11:01:45 +0800
Subject: [PATCH] ARM64: dts: rockchip: fix usb_otg compatible for
 rk3368-android

Both dwc2 and dwc_otg_310 driver are existed in the system since
commit c3c4fa0, and we use dwc_otg_310 driver for RK3368 android
platform, so only keep one compatible to fit for the driver.

Change-Id: Iee001ef057dcc63b723faf21fec94d5a4b592868
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368-android.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3368-android.dtsi
index c266bdf214e9..4962c6c10c1f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368-android.dtsi
@@ -256,6 +256,8 @@
 
 &usb_otg {
 	status = "okay";
+
+	compatible = "rockchip,rk3368-usb";
 	clocks = <&cru SCLK_OTGPHY0>, <&cru HCLK_OTG0>;
 	clock-names = "sclk_otgphy0", "otg";
 	resets = <&cru SRST_USBOTG_AHB>,
-- 
2.35.3

